zone/command.cpp

```diff
@@ -1610,7 +1610,7 @@ void command_list(Client *c, const Seperator *sep)
 					"#goto %.0f %0.f %.0f",
 					entity->GetX(),
 					entity->GetY(),
-					entity->GetZ());
+					entity->GetZ() + (entity->IsBoat() * 50));
```

Comment:
Fixed the Bool thing using a ?: instead.

Regarding the 50, I thought about this and I don't see a clear better path.   I know if narrowed too much, it will fail depending on flymode.

Personally I see this as a quick hack to fix a spot that doesn't work.  But if someone has an exact way to come up with a reliable offset based on flymode, and size or any other variable, I'm happy to use it.

Reply:
I'm fine with the ternary route, at least a bit more readable.

As far as the magic number, will 50 work for all boats? Lots of varying sizes of boats

Reply:
As far as I know, there's only one of the boat that is solid and players don't fall through?  Race 533?  I was actually using 100 and I ended up falling from the sails.  I dropped it to 50 and it was less janky, but still fell.  I guess I must be wrong and there are other boat races used as NPCs that work?  If someone has a better idea, I'll use it.

Reply:
Man github confuses me so much with updates.  I often click show changes, and I get the old code.  Maybe Chrome caching.. just never seems to just work.

Reply:
Let's roll with it for now. It's better than what it is today but we should figure out something sensible if someone is messing with it again